services/web/client/source/class/osparc/desktop/WorkbenchView.js

```diff
@@ -535,7 +535,7 @@ qx.Class.define("osparc.desktop.WorkbenchView", {
           if (node && nodeData) {
             node.setOutputData(nodeData.outputs);
             if ("state" in nodeData && node.isComputational()) {
-              node.getStatus().setRunningStatus(nodeData["state"]);
+              node.getStatus().setRunningStatus(nodeData["state"]["currentStatus"]);
```

Comment:
this is really going to look good! do you have screenshots of the UI to put in the PR?

Reply:
this comes with the frontend PR from @odeimaiz